sources/packages/web/src/components/common/ProgramDetails.vue

```diff
@@ -12,6 +13,7 @@
         variant="outlined"
         @click="programButtonAction"
         color="primary"
+        data-cy="programEditButton"
```

Comment:
for this component's context, data-cy must be programButtonAction because this button acts as edit for institution and view for ministry.

Reply:
updated as needed